### PR TITLE
Add docs to anonymized Yandex.Metrika datasets

### PR DESCRIPTION
**Pull request by alesapin**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

For changelog. Remove if this is non-significant change.

Category (leave one):
- Build/Testing/Packaging Improvement

Short description (up to few sentences):
Add docs to anonymized Yandex.Metrika datasets

### CODE REVIEW

File: docs/en/getting_started/example_datasets/metrika.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Anonymized Yandex.Metrika Data
```
Comment (blinkov) [inline docs/en/getting_started/example_datasets/metrika.md]:
```
blinkov@blinkov-1osx ~/ClickHouse/docs/en
 % ya grep Yandex.Metrika 2>/dev/null | wc -l
0
blinkov@blinkov-1osx ~/ClickHouse/docs/en
 % ya grep Yandex.Metrica 2>/dev/null | wc -l
34
```

File: docs/en/getting_started/example_datasets/metrika.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Dataset consists of two tables containing anonymized data about hits (`hits_v1`) and visits (`visits_v1`) of Yandex.Metrika. Each of the tables can be downloaded as a compressed `tsv.xz` file or as prepared partitions.
+
+## Receiving tables from prepared partitions
```
Comment (blinkov) [inline docs/en/getting_started/example_datasets/metrika.md]: * I think "obtaining" would sound better
* title case: ... Tables from Prepared Partitions

File: docs/en/getting_started/example_datasets/metrika.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+# import data
+cat hits_v1.tsv | clickhouse-client --query "INSERT INTO datasets.hits_v1 FORMAT TSV"
+# optionally you can optimize table
+clickhouse-client --query "OPTIMIZE TABLE datasets.hits_v1 FINAL"
+clickhouse-client --query "SELECT COUNT(*) FROM datasets.hits_v1"
+```
+
+**Dowload and import visits from compressed tsv-file**
```
Comment (blinkov) [inline docs/en/getting_started/example_datasets/metrika.md]: Dow**n**load

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/toc_en.yml [hunk 4]
```diff
@@ -16,7 +16,7 @@ nav:
     - 'WikiStat': 'getting_started/example_datasets/wikistat.md'
     - 'Terabyte Click Logs from Criteo': 'getting_started/example_datasets/criteo.md'
     - 'Star Schema Benchmark': 'getting_started/example_datasets/star_schema.md'
-    - 'Yandex.Metrika Data': 'getting_started/example_datasets/metrika.md'
+    - 'Yandex.Metrica Data': 'getting_started/example_datasets/metrika.md'
```
Comment (blinkov) [inline docs/toc_en.yml]: filename too then

File: docs/en/getting_started/example_datasets/metrica.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
...tentLoadedTiming Int32,  DOMCompleteTiming Int32,  LoadEventStartTiming Int32,  LoadEventEndTiming Int32,  NSToDOMContentLoadedTiming Int32,  FirstPaintTiming Int32,  RedirectCount Int8,  SocialSourceNetworkID UInt8,  SocialSourcePage String,  ParamPrice Int64,  ParamOrderID String,  ParamCurrency FixedString(3),  ParamCurrencyID UInt16,  GoalsReached Array(UInt32),  OpenstatServiceName String,  OpenstatCampaignID String,  OpenstatAdID String,  OpenstatSourceID String,  UTMSource String,  UTMMedium String,  UTMCampaign String,  UTMContent String,  UTMTerm String,  FromTag String,  HasGCLID UInt8,  RefererHash UInt64,  URLHash UInt64,  CLID UInt32,  YCLID UInt64,  ShareService String,  ShareURL String,  ShareTitle String,  \`ParsedParams.Key1\` Array(String),  \`ParsedParams.Key2\` Array(String),  \`ParsedParams.Key3\` Array(String),  \`ParsedParams.Key4\` Array(String),  \`ParsedParams.Key5\` Array(String),  \`ParsedParams.ValueDouble\` Array(Float64),  IslandID FixedString(16),  RequestNum UInt32,  RequestTry UInt8) ENGINE = MergeTree() PARTITION BY toYYYYMM(EventDate) ORDER BY (CounterID, EventDate, intHash32(UserID)) SAMPLE BY intHash32(UserID) SETTINGS index_granularity = 8192"
```
Comment (alexey-milovidov) [inline docs/en/getting_started/example_datasets/metrica.md]: Let's write Nested data structures as Nested.

File: docs/en/getting_started/example_datasets/metrica.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+# import data
+cat hits_v1.tsv | clickhouse-client --query "INSERT INTO datasets.hits_v1 FORMAT TSV"
```
Comment (alexey-milovidov) [inline docs/en/getting_started/example_datasets/metrica.md]: Let's add `--max_insert_block_size=100000` to limit memory consumption.
(it's required for servers with 4 GB RAM or less)

File: docs/en/getting_started/example_datasets/metrica.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+# import data
+cat visits_v1.tsv | clickhouse-client --query "INSERT INTO datasets.visits_v1 FORMAT TSV"
+# optionally you can optimize table
+clickhouse-client --query "OPTIMIZE TABLE datasets.visits_v1 FINAL"
+clickhouse-client --query "SELECT COUNT(*) FROM datasets.visits_v1"
+```
+
+## Queries
+Examples of queries to these tables can be found among [stateful tests](https://github.com/yandex/ClickHouse/tree/master/dbms/tests/queries/1_stateful) and in some [performance tests](https://github.com/yandex/ClickHouse/tree/master/dbms/tests/performance/test_hits) of ClickHouse.
```
Comment (alexey-milovidov) [inline docs/en/getting_started/example_datasets/metrica.md]: But these queries refer tables as `test.hits`, `test.visits`.

Comment (alesapin) [reply docs/en/getting_started/example_datasets/metrica.md]: I add note about it. Is it ok?

Comment (alexey-milovidov) [reply docs/en/getting_started/example_datasets/metrica.md]: Ок.

**Comment by blinkov** [comment]
No

**Pull request merged**